build.gradle.kts

```diff
@@ -43,14 +43,19 @@ dependencies {
   testImplementation("org.fluentlenium:fluentlenium-junit-jupiter:4.8.0")
   testImplementation("org.fluentlenium:fluentlenium-assertj:4.8.0")
   testImplementation("org.mockito:mockito-inline:4.5.1")
-  testImplementation("org.seleniumhq.selenium:selenium-java:4.1.4")
+  testImplementation("org.seleniumhq.selenium:htmlunit-driver:2.61.0")
```

Comment:
This was a rather tricky one.  After moving to a later version of Spring (via the gradle-spring-boot plugin) it played havoc with dependency conflicts.